src/dos/drive_iso.cpp

```diff
@@ -114,19 +114,20 @@ bool isoFile::Write(uint8_t* /*data*/, uint16_t* /*size*/) {
 }
 
 bool isoFile::Seek(uint32_t *pos, uint32_t type) {
+
+	uint32_t requested_pos = {};
+
 	switch (type) {
-		case DOS_SEEK_SET:
-			filePos = fileBegin + *pos;
-			break;
-		case DOS_SEEK_CUR:
-			filePos += *pos;
-			break;
-		case DOS_SEEK_END:
-			filePos = fileEnd + *pos;
-			break;
-		default:
-			return false;
+	case DOS_SEEK_SET: requested_pos = fileBegin + *pos; break;
+	case DOS_SEEK_CUR: requested_pos = filePos + *pos; break;
+	case DOS_SEEK_END: requested_pos = fileEnd + *pos; break;
+	default: return false;
 	}
+
+	if (filePos != requested_pos) {
+		filePos = requested_pos;
+	}
```

Comment:
Same comment as before.  Here I don't even see what adding a new `requested_pos` variable does.  Behavior looks to me to be exactly the same as just setting `filePos` directly like it was before.